src/chrome/content/rules/Embedresponsively.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+    Non-functional hosts:
+       Timeout:
+       - ssh.embedresponsively.com
```

Comment:
I get invalid certificate here as well, not a time out. Could you double check please?

Reply:
Hmm I got a time out last I check, but got a different response from the peer now. Adjusted comment.